scripts/globals/magic_utils/spell_damage.lua

```diff
@@ -0,0 +1,883 @@
+    [xi.magic.spell.THUNDER_III ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC, xi.magic.element.LIGHTNING,  345,  1.5,  350,  2.5,  2.5, 1.75,     1,    0,    0,    0 },
+    [xi.magic.spell.THUNDER_IV  ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC, xi.magic.element.LIGHTNING,  541,    2,  600,  3.6,  3.5, 2.75,  1.95,    1,    0,    0 },
+    [xi.magic.spell.THUNDER_V   ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC, xi.magic.element.LIGHTNING,  874,  2.3,  900,    4, 3.74, 3.75,  2.95, 1.95,    1,    0 },
+    [xi.magic.spell.THUNDER_VI  ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC, xi.magic.element.LIGHTNING, 1250,  2.5, 1250,  4.5,  5.5,  4.5,   3.5, 2.75, 1.95,    1 },
+    [xi.magic.spell.BURST       ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC, xi.magic.element.LIGHTNING,  552,    2,  700,    2,    2,    2,     2,    2,    2,    2 },
+    [xi.magic.spell.BURST_II    ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC, xi.magic.element.LIGHTNING,  710,    2,  800,    2,    2,    2,     2,    2,    2,    2 },
+    [xi.magic.spell.WATER       ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,     xi.magic.element.WATER,   16,    1,   25,  1.8,    1,    0,     0,    0,    0,    0 },
+    [xi.magic.spell.WATER_II    ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,     xi.magic.element.WATER,   95,    1,  120,  2.8,  1.9,    1,     0,    0,    0,    0 },
+    [xi.magic.spell.WATER_III   ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,     xi.magic.element.WATER,  236,  1.5,  230,  3.7,  2.9, 1.95,     1,    0,    0,    0 },
+    [xi.magic.spell.WATER_IV    ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,     xi.magic.element.WATER,  410,    2,  440,  4.7,  3.9, 2.95,  1.99,    1,    0,    0 },
+    [xi.magic.spell.WATER_V     ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,     xi.magic.element.WATER,  680,  2.3,  700,  5.6, 4.74, 3.95,  2.99, 1.99,    1,    0 },
+    [xi.magic.spell.WATER_VI    ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,     xi.magic.element.WATER, 1010,  1.5, 1010,  6.5,  5.9,  4.9,   3.9, 2.95, 1.99,    1 },
+    [xi.magic.spell.FLOOD       ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,     xi.magic.element.WATER,  552,    2,  700,    2,    2,    2,     2,    2,    2,    2 },
+    [xi.magic.spell.FLOOD_II    ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,     xi.magic.element.WATER,  710,    2,  800,    2,    2,    2,     2,    2,    2,    2 },
+    [xi.magic.spell.COMET       ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,      xi.magic.element.DARK,  964,  2.3, 1000,    4, 3.75,  3.5,     3,    2,    1,    1 },
+
+-- Multiple target spells:
+-- Structure:           [spellId] = {  Stat used,                    Skill,                    Element, vNPC, mNPC,  vPC,   M0,  M50,  M100, M200, M300, M400, M500 },
+    [xi.magic.spell.AEROGA      ] = { xi.mod.INT, xi.skill.ELEMENTAL_MAGIC,      xi.magic.element.WIND,   93,    1,  100,  2.6,  1.8,     1,    0,    0,    0,    0 },
```

Comment:
Maybe we could make these liens less wide by casting some things to variables or by getting some things from other locations rather than having to repeat them into the table/array

like
```lua
local INT = xi.mod.INT
local elemSkill = xi.skill.ELEMENTAL_MAGIC
```
```lua
[xi.magic.spell.AEROGA      ] = { INT, elemSklill,      xi.magic.element.WIND,   93,    1,  100,  2.6,  1.8,     1,    0,    0,    0,    0 },
```
or we use `spell:getSkillType()` to not need a table entry at all

Reply:
we can "get" the element as well.

Reply:
The reason I defined elements into the table was to make less calls to external functions. I'm guessing it is making a call to the database.
I was hoping that someone (not me, someone that actually knows how to do it) would compare both methods and check wich one was more eficient resource wise.
It also helped me when filling the tables. A visual help, so to speak.

I'm not oposed to creating local vars to shorten the length though.

Reply:
I see Teo has already had the same idea as me...